deployment/ccip/changeset/testhelpers/test_sui_helpers.go

```diff
@@ -230,12 +231,12 @@ func SendSuiCCIPRequest(e cldf.Environment, cfg *ccipclient.CCIPSendReqConfig) (
 
 		// For SUI -> EVM BurnMint Pool token Transfer, we can use msg.Receiver as tokenReceiver, this field is only used in usdc token pool
 		// bc we need to check the recipient with Circle's packages from the onramp side before sending USDC. and it's not used anyway else.
-		decodedTokenReceiver, _ := hex.DecodeString("0000000000000000000000000000000000000000000000000000000000000000")
-		var tokenReceiver [32]byte
-		copy(tokenReceiver[:], decodedTokenReceiver)
+		// decodedTokenReceiver, _ := hex.DecodeString("0000000000000000000000000000000000000000000000000000000000000000")
+		// var tokenReceiver [32]byte
+		// copy(tokenReceiver[:], decodedTokenReceiver)
 
 		paramValues := []any{
-			decodedTokenReceiver,
+			msg.TokenReceiverATA,
```

Comment:
we probably have to change existing ccip tests inputs then 

Also in what cases will msg.Reciever != tokenReciever?

Reply:
yes. but when it's a pure token transfer, we should not set msg.receiver, right? we should only set token receiver in extra args in that case.